NuGet.config

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <packageSources>
-    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
-    <clear />
-    <add key="BlobFeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="nugetbuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
-    <add key="msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
-    <add key="dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
-    <add key="dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/api/v3/index.json" />
-    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-    <add key="roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
-    <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
-    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-  <config>
-    <add key="repositoryPath" value="packages" />
-    <add key="globalPackagesFolder" value="packages" />
-  </config>
-</configuration>
```

Comment:
Should we still have one with <clear /> to ensure that nothing is just using the machine config?

Test assets won't import Common.props, so I suspect that's what tests are doing now.

Reply:
Put back a NuGet.config with just the 'config' values.

Reply:
My comment got rendered wrong (edited). I think we should have a `<clear/>` here to ensure that things are actually using RestoreSources and not the machine/default config. I suspect tests are doing the latter.